bartender/_ssh_utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional, TypedDict
+
+from asyncssh import SSHClientConnection, connect
+
+
+class SshConnectConfig(TypedDict):
+    """Configuration for ssh connection."""
+
+    hostname: str
+    port: Optional[int]
+    username: Optional[str]
+    password: Optional[str]
+
+
+async def ssh_connect(config: SshConnectConfig) -> SSHClientConnection:
+    """Connect to a host using SSH.
+
+    :param config: Configuration.
+    :return: The connection.
+    """
+    conn_vargs = {
+        "known_hosts": None,
+    }
+    if config["password"] is not None:
+        conn_vargs["agent_path"] = None
```

Comment:
I guess setting `known_hosts` to None ignores the confirmation prompt for new connections. What does the `agent_path` setting do?

Reply:
At https://github.com/i-VRESSE/bartender/blob/3bd16de1a2ed0a338067aeee97faf6ea8a1a75e2/src/bartender/ssh_utils.py#L32 has been corrected to not use ssh-agent when a password is supplied.